spec/services/notification_service_spec.rb

```diff
@@ -187,29 +187,54 @@ def should_not_email(user_id, n)
   end
 
   describe 'Issues' do
-    let(:issue) { create :issue, assignee: create(:user), description: 'cc @participant' }
+    let(:issue) { create :issue, assignee: create(:user),
+                  description: description_text }
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Expression at 191, 49 should be on its own line.